indra/newview/lltoolpie.h

```diff
@@ -98,15 +98,12 @@ class LLToolPie : public LLTool, public LLSingleton<LLToolPie>
         MEDIA_FIRST_CLICK_FRIEND     = 1 << 3,    // 0b00001000 (8)
         MEDIA_FIRST_CLICK_LAND       = 1 << 4,    // 0b00010000 (16)
 
-        // Covers any object with PRIM_MEDIA_FIRST_CLICK_INTERACT (combines all other flags)
-        MEDIA_FIRST_CLICK_ANY        = MEDIA_FIRST_CLICK_HUD &
-                                       MEDIA_FIRST_CLICK_OWN &
-                                       MEDIA_FIRST_CLICK_GROUP &
-                                       MEDIA_FIRST_CLICK_FRIEND &
-                                       MEDIA_FIRST_CLICK_LAND,
+        // Covers any object with PRIM_MEDIA_FIRST_CLICK_INTERACT (combines all previous flags)
+        MEDIA_FIRST_CLICK_ANY        = 1073741823,// 0b00111111111111111111111111111111
```

Comment:
I'm not entirely a fan of a magic number like this.  Looking below I understand why however.  There should be a better way to do this without it though.